internal/system/agent/v2/application/executor/operation.go

```diff
@@ -0,0 +1,60 @@
+//
+// Copyright (C) 2021 IOTech Ltd
+//
+// SPDX-License-Identifier: Apache-2.0
+
+package executor
+
+import (
+	"context"
+	"net/http"
+	"sync"
+
+	"github.com/edgexfoundry/go-mod-core-contracts/v2/clients/logger"
+	"github.com/edgexfoundry/go-mod-core-contracts/v2/errors"
+	"github.com/edgexfoundry/go-mod-core-contracts/v2/v2/dtos/common"
+	"github.com/edgexfoundry/go-mod-core-contracts/v2/v2/dtos/requests"
+
+	"github.com/edgexfoundry/edgex-go/internal/system/agent/interfaces"
+)
+
+// operation contains references to dependencies required to handle the operation via external executor use case.
+type operation struct {
+	executor     interfaces.CommandExecutor
+	executorPath string
+	lc           logger.LoggingClient
+}
+
+// NewOperation is a factory function that returns an initialized operation receiver struct.
+func NewOperation(executor interfaces.CommandExecutor, executorPath string, lc logger.LoggingClient) *operation {
+	return &operation{
+		executor:     executor,
+		executorPath: executorPath,
+		lc:           lc,
+	}
+}
+
+// Do implements the Metrics interface to obtain metrics via executor for one or more services concurrently.
```

Comment:
I don't think this comment correct, i.e. specific to metrics? Please re-word the comment.